ui_framework/src/components/page/_page_content.scss

```diff
@@ -0,0 +1,16 @@
+.kuiPageContent {
+  padding: $kuiSizeL;
+  background: $kuiColorEmptyShade;
+  border: $kuiBorderThin;
+  border-radius: $kuiBorderRadius;
+  width: 100%;
+
+  &.kuiPageContent--verticalCenter {
+    align-self: center;
+  }
+
+  &.kuiPageContent--horizontalCenter {
+    width: auto;
+    margin: auto;
+  }
```

Comment:
Can we unnest these modifiers to avoid the added specificity?

```scss
.kuiPageContent {
  padding: $kuiSizeL;
  background: $kuiColorEmptyShade;
  border: $kuiBorderThin;
  border-radius: $kuiBorderRadius;
  width: 100%;
}

.kuiPageContent--verticalCenter {
  align-self: center;
}

.kuiPageContent--horizontalCenter {
  width: auto;
  margin: auto;
}
```